index.d.ts

```diff
@@ -0,0 +1,287 @@
+/// <reference types="node" />
+
+/**
+ * Elegant terminal spinner.
+ *
+ * @param options If a string is provided, it is treated as a shortcut for `options.text`.
+ */
+export default function ora(options?: Options | string): Ora;
+
+/**
+ * Starts a spinner for a promise. The spinner is stopped with `.succeed()` if the promise fulfills
+ * or with `.fail()` if it rejects.
+ *
+ * @param action
+ * @param options If a string is provided, it is treated as a shortcut for `options.text`.
+ * @returns The spinner instance.
+ */
+export function promise(action: PromiseLike<any>, options?: Options | string): Ora;
```

Comment:
Use `unknown`.

Reply:
Not relevant here and better left as `any`. This interface accepts any input, not some input. `unknown` would be relevant if we would return this value to the user.

Reply:
`unknown` works exactly the same as `any` here. I prefer using `unknown` for consistency.

Reply:
What do you think about this: https://github.com/sindresorhus/p-event/pull/21#discussion_r260744213 ?